code/game/gamemodes/management/management.dm

```diff
@@ -73,3 +73,25 @@
 		ABNORMALITY_ORIGIN_LIMBUS,
 		ABNORMALITY_ORIGIN_ORIGINAL
 		)
+
+
+/datum/game_mode/management/joke
+	name = "L Corp - Freaky Friday"
+	config_tag = "jokemode"
+	votable = 0
+
+	announce_span = "notice"
+	announce_text = "Manage all abnormalities not originating from Lobotomy Corporation HQ, as well as a selection of joke and reference abnormalities."
```

Comment:
This is not accurate as `ORIGIN_LOBOTOMY` is kept. Update one of the two please.